jekyll/_sass/_sidebar.scss

```diff
@@ -50,7 +50,6 @@ $sidebar-spacing-base: 5px;
 
   &.fixed {
     position: fixed;
-    top: 118px;
```

Comment:
Fixed position won't have an impact without a `top` value.

Reply:
It does; it uses whatever the natural (statically-positioned) position is, which is what we want in this case.